<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>나눔고딕</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>581</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>정보입력</string>
       </property>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>26</y>
          <width>60</width>
          <height>12</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>나눔고딕</family>
         </font>
        </property>
        <property name="text">
         <string>이      름 :</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLineEdit" name="txtName">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>20</y>
          <width>210</width>
          <height>20</height>
         </rect>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhDigitsOnly|Qt::ImhLatinOnly</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>86</y>
          <width>60</width>
          <height>12</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>나눔고딕</family>
         </font>
        </property>
        <property name="text">
         <string>이 메 일 :</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLineEdit" name="txtEmail">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>80</y>
          <width>210</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>56</y>
          <width>60</width>
          <height>12</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>나눔고딕</family>
         </font>
        </property>
        <property name="text">
         <string>전화번호 :</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLineEdit" name="txtPhone">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>50</y>
          <width>210</width>
          <height>20</height>
         </rect>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhFormattedNumbersOnly</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>116</y>
          <width>60</width>
          <height>12</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>나눔고딕</family>
         </font>
        </property>
        <property name="text">
         <string>주      소 :</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLineEdit" name="txtAddress">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>110</y>
          <width>210</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="btnNew">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>100</y>
          <width>80</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string>신규</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnSave">
        <property name="geometry">
         <rect>
          <x>398</x>
          <y>100</y>
          <width>80</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string>저장</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnDel">
        <property name="geometry">
         <rect>
          <x>485</x>
          <y>100</y>
          <width>80</width>
          <height>30</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgb(255,50,50);
color:rgb(255,255,255);</string>
        </property>
        <property name="text">
         <string>삭제</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>159</y>
      <width>581</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTableWidget" name="tblAddress"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="stbCurrent"/>
 </widget>
 <resources/>
 <connections/>
</ui>
